fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -19,11 +19,17 @@ def self.handle(args, options)
       end
 
       platform = nil
-      lane = platform_lane_info[1]
```

Comment:
This code block is a bit complicated, maybe we could simplify it to something like 

```ruby
 platform, lane = if platform_lane_info.count > 1
                    platform_lane_info.shift(2)
                  else
                    [nil, platform_lane_info.shift]
                  end
```

Maybe it's just me but I was honestly kind of confused with the ordering of the`delete_at`s and that `nil`-dodging `if`

Reply:
Probably because it evolved out of what was there before.

Reply:
Yes indeed, this causes it to try to execute a `foo bool` lane, where `foo` is the platform and `bool` the lane name :(